src/scripts/fields/geoLocation.coffee

```diff
@@ -4,27 +4,14 @@ Formbuilder.registerField 'geolocation',
 
   order: 60
 
-  element_type: 'non_input'
-
   view: """
-    <label class='section-name'><%- rf.get(Formbuilder.options.mappings.LABEL) %></label>
     <div class="form-group">
      <button class='fb-button'>Get GeoLocation</button>
     </div>
-    <p><%- rf.get(Formbuilder.options.mappings.DESCRIPTION) %></p>
-
     <span><%=rf.geolocationFunctionality%></span>
   """
 
   edit: """
```

Comment:
Do we need remove the description from geolocation?

Reply:
Yes! What I discovered was that when a formbuilder field is given the type non_input, a bunch of automatic styling including the label, description <p>, and the 'required' asterisk are removed. For this reason, the person who implemented the geolocation and gave it a non_input type had to manually add this stuff in. Now that I'm making it a regular input field again, I needed to remove the extra styling, or else the geolocation box ends up with two labels and two descriptions..!